### ISSUE TITLE
xfer2 glitch on cs.

### ISSUE THREAD

**Issue by ead0601** [doceme/py-spidev#55]
Hi,

I am using a Pi 3 version 8 (jessie), and py-spidev (HEAD) (Merge: 511db0d 11adf8c)

When I execute:

resp = spi.xfer2([0xD8, 0x00, 0x00, 0x00])

I see a glitch on CS during the transfer.

Thanks,
Edward

**Comment by ead0601** [comment]
I wanted to add that the glitch occurs near the begining of the transfer, and for the rest of the burst of 4 bytes, the glitch is not there.

**Comment by ead0601** [comment]
I added the following to /boot/config.txt

core_freq=250

This disables the auto frequency throttling, and fixes the core frequency. 
   1) I no longer see the glitch.
   2) However, very intermittently, I receive the incorrect data. I have slowed down the transfer and ensured proper grounds, and a problem still occurs. The SI looks good on a scope.

Given the improved performance with the "core_freq" setting, I can implement a retry work around. Luckily the first couple of MISO bits are fixed, so I will know when to retry.

Something, is still not Kosher.

Edward
